applications/crossbar/src/crossbar_view.erl

```diff
@@ -234,6 +235,7 @@ build_load_range_params(Context, View, Options) ->
                                      ,start_time => StartTime
                                      ,view_options => build_view_query(Options, Direction, StartKey, EndKey, HasQSFilter)
                                      ,has_qs_filter => HasQSFilter
+                                     ,start_key => StartKey
```

Comment:
@icehess  not related but, can you fix the duplicated `has_qs_filter` assignment ?